src/main/java/TaskManager.java

```diff
@@ -0,0 +1,151 @@
+import java.util.Scanner;
+
+public class TaskManager {
+
+    /**
+     * Prints a line on the console
+     */
+    public static void printLine() {
+        System.out.println("____________________________________________________________");
+    }
+
+    /**
+     * Determines the task type,
+     * assuming it is the first
+     * word in the message and
+     * returns it
+     *
+     * @param message The input by the user.
+     * @return The first word of the string.
+     */
+    private static String taskType(String message) {
+        String[] type = message.split(" ");
+        return type[0];
+    }
+
+
+    /**
+     * The function adds the task input by the user
+     *
+     * @param tasks   the array of tasks
+     * @param message the input string containing
+     *                the task description
+     */
+    private static void addTask(Task[] tasks, String message) {
+        switch (taskType(message)) {
+        case "todo":
+            addTodo(tasks, message);
+            break;
+        case "deadline":
+            addDeadline(tasks, message);
+            break;
+        case "event":
+            addEvent(tasks, message);
+            break;
+        default:
```

Comment:
Instead of leaving it blank for default, I think you could add an error method here instead.